app/Observers/PostObserver.php

```diff
@@ -15,6 +15,11 @@ class PostObserver
      */
     public function creating(Post $post)
     {
+        // If the post's signup has a group_id, save it on the post as well
+        if ($post->signup->group_id) {
```

Comment:
I think we might want an additional check in this conditional to only set the `post->group_id` if it hasn't been set yet. The chompy import is an example where we create the post with a `group_id` parameter, and the signup wouldn't have existed before creating the post.

Reply:
I added a test for that scenario that passed without any changes, but I added the check for existing `post->group_id` in anyway just to be safe!